### ISSUE TITLE
No battery status 

### ISSUE THREAD

**Issue by Maxgarnix** [TA2k/ioBroker.bosch-ebike#14]
Hello everyone,
I think it is essential to know the battery status of your bike.
Do you see a change to implement this?
 I use the Flow app.
Many thanks and best regards

**Comment by TA2k** [comment]
Is this information available in the app

**Comment by Maxgarnix** [comment]
Yes, right on the main page.
![Screenshot_20240111_075428_Flow](https://github.com/TA2k/ioBroker.bosch-ebike/assets/66115974/4aa15594-0bb5-4f6e-b6d4-b05680a9134a)

**Comment by TA2k** [comment]
Ist this only available via bluetooth.
This means when you delete the app and reinstall and never connect with your bike you can still see the value?

**Comment by Maxgarnix** [comment]
Yes, the battery status is only visible after the Bluetooth connection to the bike.  Only the trip data comes via the APP.

**Comment by TA2k** [comment]
Adapter can only fetch data without bluetooth

**Comment by StevieC121176** [comment]
> Yes, the battery status is only visible after the Bluetooth connection to the bike. Only the trip data comes via the APP.

hier scheint sich etwas geändert zu haben. Jetzt kann der aktuelle Akkustand auch über das Connect Modul von unterwegs aus abgefragt werden und es werden Daten in der Cloud gespeichert, nicht nur auf dem Smartphone. Evtl. hat sich die API geändert und der Ladestand kann jetzt auch darüber abgefragt werden?
